pkgs/applications/window-managers/hyprwm/hyprland/default.nix

```diff
@@ -41,16 +42,18 @@
 }:
 assert lib.assertMsg (!nvidiaPatches) "The option `nvidiaPatches` has been removed.";
 assert lib.assertMsg (!enableNvidiaPatches) "The option `enableNvidiaPatches` has been removed.";
-assert lib.assertMsg (!hidpiXWayland) "The option `hidpiXWayland` has been removed. Please refer https://wiki.hyprland.org/Configuring/XWayland";
+assert lib.assertMsg (!hidpiXWayland) "The option `hidpiXWayland` has been removed. Please refer https://wiki.hyprland.org/Configuring/XWayland"; let
+  wlr = wlroots.override { inherit enableXWayland; };
```

Comment:
Is this "internal overriding" a good idea?

Reply:
It might cause rebuilds for people not wanting to use XWayland, but I don't see a simpler way of providing an xwayland-free environment.

Reply:
Hum, understood. That's OK then.

Reply:
We can change the argument name to `wlroots-hyprland`. Same for `udis86`.

Reply:
> We can change the argument name to `wlroots-hyprland`. Same for `udis86`.

That's better!